tex/solutions.tex

```diff
@@ -371,6 +371,23 @@ \section{Solutions}
 This category also can not have an initial object, indeed: There is no morphism from $y$ to $x$, hence $y$ can not be initial. But also $x$ can not be initial since $f$ and $g$ are different morphisms.
 \end{itemize}
 \end{solution}
+
+\begin{solution}[\cref{exer:initial_pointset}]\label{sol:initial_pointset}
+	An initial object is a one-element set $ (\{ \star \}, \star) $. Let $ (X, x) $ be a pointed set. Then we have a unique function $ \star \to X $ that sends $ \star $, the chosen (and only) point of $ \{ \star \} $, to $ x $, the chosen point of $ X $, namely $ f: \star \mapsto x $.
+\end{solution}
+
+\begin{solution}[\cref{exer:terminal_set}]\label{sol:terminal_set}
+	A terminal set in the category of sets is a one-element set $ \{ \star \} $. Given any set $ X $, we have a unique function $ X \to \star $, since any element of $ X $ must be sent to $ \star $.
+\end{solution}
+
+\begin{solution}[\cref{exer:terminal_posetcat}]\label{sol:terminal_posetcat}
+	Given a poset $ (X, \leq) $, a terminal object in the category $ \POS(X, \leq) $ would correspond to a maximal element in $ X $. Indeed, given such a maximal element $ x \in X = \POS(X, \leq)_0 $, we have for all $ y \in \POS(X, \leq)_0 $, since $ x $ is maximal, that $ y \leq x $. Therefore, we have a morphism $ f: x \to y $. By the definition of the hom-sets in $ \POS(X, \leq) $, $ f $ is unique and we conclude that $ x $ is a terminal object.
```

Comment:
```suggestion
	Given a poset $ (X, \leq) $, a terminal object in the category $ \POS(X, \leq) $ is exactly a maximal element in $ X $. Indeed, given such a maximal element $ x \in X = \POS(X, \leq)_0 $, we have for all $ y \in \POS(X, \leq)_0 $, since $ x $ is maximal, that $ y \leq x $. Therefore, we have a morphism $ f: x \to y $. By the definition of the hom-sets in $ \POS(X, \leq) $, $ f $ is unique and we conclude that $ x $ is a terminal object.
	Conversely, unfolding the definition of terminal object shows that a terminal object yields a maximal element.
```